models/classes/Lists/Business/Domain/DependsOnPropertyCollection.php

```diff
@@ -22,8 +22,20 @@
 
 namespace oat\tao\model\Lists\Business\Domain;
 
+use JsonSerializable;
 use oat\generis\model\resource\DependsOnPropertyCollection as GenerisDependsOnPropertyCollection;
 
-class DependsOnPropertyCollection extends GenerisDependsOnPropertyCollection
+class DependsOnPropertyCollection extends GenerisDependsOnPropertyCollection implements JsonSerializable
```

Comment:
Lets remove the usage of `use oat\generis\model\resource\DependsOnPropertyCollection as GenerisDependsOnPropertyCollection;`

```suggestion
class DependsOnPropertyCollection extends ArrayIterator implements JsonSerializable
```

Reply:
Please do not forget this change.